src/Http/Response.php

```diff
@@ -684,7 +441,7 @@ public function getReasonPhrase(): string
      */
     public function setTypeMap(string $type, array|string $mimeType): void
     {
-        $this->_mimeTypes[$type] = $mimeType;
```

Comment:
I'll add this to the migration notes. While we haven't changed any public APIs we could trip up application code with this change.